exporter/exporter.go

```diff
@@ -171,8 +171,8 @@ func (exporter *NebulaExporter) collect(wg *sync.WaitGroup, namespace, clusterNa
 		defer wg.Done()
 		statusMetrics, err := getNebulaComponentStatus(podIpAddress, podHttpPort)
 		if err != nil {
+			statusMetrics = []string{"count=0"}
```

Comment:
You have handled metrics in `getNebulaComponentStatus`, it might be better to it handled ​ it  in  one  place .

rename  `getNebulaComponentStatus`  to  `isNebulaComponentRunning`?